cardano-db-sync/src/Cardano/DbSync/Era/Shelley/OffChain.hs

```diff
@@ -0,0 +1,154 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
Could this use 
```
git mv cardano-db-sync/src/Cardano/DbSync/Era/Shelley/OffChain.hs
```
to retain file history?

Reply:
I'm not sure this would have the desired effect. 

> Git has a rename command git mv, but that is just for convenience. The effect is indistinguishable from removing the file and adding another with different name and the same content. 

More info here: https://archive.kernel.org/oldwiki/git.wiki.kernel.org/index.php/GitFaq.html#Why_does_Git_not_.22track.22_renames.3F

Reply:
Indeed, but in addition a separate "move files" commit and then an "edit file" commit would help git track the change better.

Reply:
It didn't work if you check the git history, it shows in different commits the file was reverted -> renamed -> edited but github seems to pick it up as a brand new file 🤷